internal/utils/copy_from_image.go

```diff
@@ -38,6 +39,57 @@ func CopyFromImage(image, srcInImage, destOnHost string) error {
 	if err != nil {
 		return fmt.Errorf("copy from image: %s: %w", stderr.String(), err)
 	}
+	excludeFiles := []string{".gitkeep", ".ini", ".env", ".DS_Store"}
+	excludeDirs := []string{".git", ".zeabur", ".vscode", ".idea", ".github"}
+	err = deleteFilesRecursively(excludeFiles, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete files in directory: %w", err)
+	}
+	err = deleteDirectories(excludeDirs, destOnHost)
+	if err != nil {
+		return fmt.Errorf("delete directories in directory: %w", err)
+	}
+
+	return nil
+}
+
+func deleteFilesRecursively(deleteFiles []string, path string) error {
+	// walk through the directory
+	err := filepath.Walk(path, func(filePath string, fileInfo os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !fileInfo.IsDir() {
+			fileName := fileInfo.Name()
 
+			for _, targetFile := range deleteFiles {
+				if fileName == targetFile {
```

Comment:
Don't nest too much.

```suggestion
				if fileName != targetFile { continue }
```